fdbserver/workloads/IncrementalBackup.actor.cpp

```diff
@@ -122,12 +122,13 @@ struct IncrementalBackupWorkload : TestWorkload {
 				    .detail("ContiguousLogEndVersion",
 				            desc.contiguousLogEnd.present() ? desc.contiguousLogEnd.get() : invalidVersion)
 				    .detail("TargetVersion", v);
-				if (self->waitRetries != -1 && tries > self->waitRetries)
```

Comment:
revert the waitRetries changes from PR#8984. it doesn't make things better or worse, so leave it as is